beard/clustering/__init__.py

```diff
@@ -11,10 +11,12 @@
 
 from .blocking import BlockClustering
 from .blocking_funcs import block_double_metaphone
+from .blocking_funcs import block_last_name_first_initial
 from .blocking_funcs import block_single
 from .wrappers import ScipyHierarchicalClustering
 
 __all__ = ("BlockClustering",
            "block_double_metaphone",
+           "block_last_name_first_initial",
```

Comment:
Thanks for this, it was missing indeed :) 

Maybe we could amend `examples/author_disambiguation.py` to make use of this?
https://github.com/inveniosoftware/beard/blob/master/examples/author_disambiguation.py#L71